tools_docker/Debian_Uwsgi/entrypoint.sh

```diff
@@ -67,8 +67,8 @@ fi
 
 chown www-data:www-data -R /var/tracim
 
-# activate apache mods
-a2enmod proxy proxy_http proxy_ajp rewrite deflate headers proxy_html dav_fs dav expires
+# activate apache modules
+a2enmod proxy proxy_http proxy_ajp rewrite deflate headers proxy_html dav_fs dav expires proxy_uwsgi
 
 # Activate or deactivate webdav
 if [ "$START_WEBDAV" = "1" ]; then
```

Comment:
I purpose to add space in:
- `sed -i "s|^\s*#ProxyPass http://127.0.0.1:3030/webdav|` => `sed -i "s|^\s*# ProxyPass http://127.0.0.1:3030/webdav|`
- `sed -i "s|^\s*#ProxyPassReverse http://127.0.0.1:3030/webdav|` => `sed -i "s|^\s*# ProxyPassReverse http://127.0.0.1:3030/webdav|`

- `|        #ProxyPass http://127.0.0.1:3030/webdav|g` => `|        # ProxyPass http://127.0.0.1:3030/webdav|g`
- `|        #ProxyPassReverse http://127.0.0.1:3030/webdav|g` => `|        # ProxyPassReverse http://127.0.0.1:3030/webdav|g`